tests-integration/src/install.rs

```diff
@@ -23,18 +23,32 @@ pub(crate) const BASE_ARGS: &[&str] = &[
     "label=disable",
 ];
 
+// ostree_ext::container::deploy::STATEROOT_DEFAULT
+const DEFAULT_STATEROOT: &str = "default";
+
+// arbitary
+const NON_DEFAULT_STATEROOT: &str = "foo";
+
 // Clear out and delete any ostree roots
 fn reset_root(sh: &Shell) -> Result<()> {
     // TODO fix https://github.com/containers/bootc/pull/137
-    if !Path::new("/ostree/deploy/default").exists() {
-        return Ok(());
+
+    for stateroot in [DEFAULT_STATEROOT, NON_DEFAULT_STATEROOT] {
+        if !Path::new(&format!("/ostree/deploy/{stateroot}")).exists() {
+            continue;
+        }
+
+        if Path::new(&format!("/ostree/deploy/{stateroot}/deploy")).exists() {
+            cmd!(
+            sh,
+            // env var hack needed because cmd! doesn't interpolate inside single quotes
```

Comment:
Worth reviewing

Reply:
Yeah, there's a lack of a "list stateroots" API in ostree here that's biting us. And a lack of an API to prune.

Hmmm...maybe in the short term we can add a hidden `bootc install wipe-ostree` or something?

Reply:
A distinct but related issue here is the test suite semi-intentionally doesn't *itself* depend on ostree-rs-ext because it creates logistical complications for us in CI scenarios so it's best if we rely on hidden CLI bits in bootc to start and then we can use our experience from CI to potentially "graduate" those CLIs to stable APIs for others later.

Reply:
>Worth reviewing: any better ideas?

My comment was referring particularly to the single quotes thing, but I opened #776 if we want to pursue your suggestion further

Reply:
Let me know if you prefer I add that subcommand in this PR instead

Reply:
Force pushed to use #777